src/EnergyPlus/VariableSpeedCoils.hh

```diff
@@ -50,7 +50,7 @@
 
 // ObjexxFCL Headers
 #include <ObjexxFCL/Array1D.fwd.hh>
-#include <ObjexxFCL/Optional.fwd.hh>
+#include <ObjexxFCL/Optional.hh>
```

Comment:
Technically I should have probably left this as the forward declaration header, but since the intent is for all of these to be gone, it just makes it simpler to just have to look for the one header in a file to know that something needs to be done there.